### PR TITLE
[Styling] Visual refresh.

### PR DESCRIPTION
**Pull request by willlarche**
screen shot 2018-02-20 at 4 57 43 pm
screen shot 2018-02-20 at 4 59 03 pm

### CODE REVIEW

File: codelab-style.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
 -->
 
 <link rel="import" href="../paper-styles/color.html">
+<link href="https://fonts.googleapis.com/css?family=Google+Sans" rel="stylesheet">
```
Comment (ebidel) [inline codelab-style.html]: This should be in an html import like https://github.com/googlecodelabs/codelab-components/blob/master/font-source-code-pro.html and then used like https://github.com/googlecodelabs/codelab-components/blob/0b8dcb6458290d02e6ff9cdc0430d9c03973bbf3/syntax-style.html#L17

File: codelab-style.html [hunk 2]
```diff
@@ -142,63 +146,81 @@
...
+
 #toc {
   /* ensures bottom of nav scrolling meny doesn't overlap with file bug link. */
   margin-bottom: 1.5em;
 }
 
 .toc-item {
   font-size: 14px;
-  color: var(--paper-grey-500);
+  color: #80868B;
```
Comment (ebidel) [inline codelab-style.html]: we should define a var for #80868B and reuse that throughout.

File: demo/codelab.html [hunk 3]
```diff
@@ -106,7 +106,7 @@ <h2>Install the <google-map> element</h2>
   "iron-elements": "PolymerElements/iron-elements#^1.0.0",
   "paper-elements": "PolymerElements/paper-elements#^1.0.1",
   "google-map": "GoogleWebComponents/google-map#^1.0.3"
-}</code></pre>
+} // test</code></pre>
```
Comment (ebidel) [inline demo/codelab.html]: what's this?

**Comment by ebidel** [comment]
Can you add screenshots of the updates?

File: demo/codelab.html (hunk 3 above)
Comment (willlarche) [reply demo/codelab.html]: A test that slipped in! Removing.

File: codelab-style.html (hunk 1 above)
Comment (willlarche) [reply codelab-style.html]: Done

File: demo/codelab.html (hunk 3 above)
Comment (willlarche) [reply demo/codelab.html]: Done

File: codelab-style.html (hunk 2 above)
Comment (willlarche) [reply codelab-style.html]: I don't know exactly how to do that...What's a good example?

Comment (ebidel) [reply codelab-style.html]: Like `--nav-height` at the top of this file.

File: codelab-style.html [hunk 4]
```diff
@@ -15,6 +15,7 @@
 -->
 
 <link rel="import" href="../paper-styles/color.html">
+<link rel="import" href="font-source-code-pro.html">
```
Comment (ebidel) [inline codelab-style.html]: Don't you want Google Sans? Create a new google-sans.html file, load it here. It's contents:

`<link href="https://fonts.googleapis.com/css?family=Google+Sans" rel="stylesheet">`

File: codelab-style.html (hunk 2 above)
Comment (willlarche) [reply codelab-style.html]: Done! For most repeated colors too.

**Comment by willlarche** [comment]
PTAL

File: codelab-style.html [hunk 5]
```diff
@@ -60,10 +73,11 @@
 
 .drawer-content-wrapper {
   padding: 16px;
+  border-right: 1px solid var(--google-material-grey-300);
 }
 
 #main-content {
-  background: var(--google-codelab-background, --paper-grey-300);
+  background: var(--google-codelab-background, --paper-blue-300);
```
Comment (ebidel) [inline codelab-style.html]: Should we leave the md v1 blues (`--paper-blue-*`?) in this file or update them too? If you update them, also remove the ` <link rel="import" href="../paper-styles/color.html">` at the top of this file.

**Comment by ebidel** [comment]
LGTM. Just the last comment.

File: codelab-style.html (hunk 5 above)
Comment (willlarche) [reply codelab-style.html]: I think that I should but it should be in a PR after this one. I didn't want to change things I couldn't immediately perceive. I might want your help on that and the minimal theme.

**Comment by willlarche** [comment]
Merge when you can. I don't have write access to this repo.

**Pull request merged**